src/components/GordonDialogBox/index.js

```diff
@@ -41,6 +41,7 @@ const GordonDialogBox = ({
   return (
     <Dialog
       className={styles.gc360_gordondialogbox}
+      onClose={cancelButtonClicked}
```

Comment:
This changes the default behavior for all GordonDialogBoxes to close whenever someone clicks outside the dialog, hits esc, or presses a close button. While I don't necessarily think that's a bad thing, I do think that it needs to be a considered and tested change, especially since some of our dialogs are multistep forms where closing unexpectedly could be quite frustrating to the user. Perhaps we could create an issue about this, but for this current pull request, I would think it's better to just pass an `onClose` function to the `GordonDialogBox` in `SpreadSheetUploader`, which will have the same effect for that dialog but won't change every other use of our dialog component site-wide.